app/src/oculusvrArmDebug/AndroidManifest.xml

```diff
@@ -8,6 +8,14 @@
     <uses-feature android:name="oculus.software.handtracking" android:required="false" />
     <uses-permission android:name="com.oculus.permission.HAND_TRACKING" />
 
+    <!-- Not needed in Android 10 (API >= 29) -->
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" tools:node="remove"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" tools:node="remove"/>
+
+    <!-- Requested by GeckoView but not needed in VR -->
+    <uses-permission android:name="android.permission.WAKE_LOCK" tools:node="remove"/>
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" tools:node="remove"/>
+
```

Comment:
Any reason why we don't do this in the main application manifest? I mean, if we don't need them for Oculus we don't likely need them for any other platform.